documentation/src/main/asciidoc/topics/con_server_hinting.adoc

```diff
@@ -11,14 +28,14 @@ Rack::
   hardware or network failure at the same time.
 Site::
   Some clusters may have nodes in multiple physical locations for extra resilience.
-  Note that Cross site replication is another alternative for
-  clusters that need to span two or more data centres.
-
-All of the above are optional.
-When provided, the distribution algorithm will try to spread the ownership of each segment across as many sites, racks, and machines (in this order) as possible.
+  Note that cross site replication is another alternative for
+  clusters that need to span two or more data centers.
```

Comment:
This is reference information that we want to move to the XSD. 

Move all the attribute descriptions here: https://github.com/infinispan/infinispan/blob/492d240fd5fe3a1f672e14e0206aa95fd63778e9/core/src/main/resources/schema/infinispan-config-14.0.xsd#L459

We should then reference the schema as part of the `Additional resources` section.